src/components/Actions/Actions.tsx

```diff
@@ -7,14 +7,14 @@ import { StyledActions, StyledActionButton, StyledActionsWrapper } from './Actio
 import { ActionsProps } from './Actions.types';
 
 export const Actions = ({ items, context, visibleByDefault = false }: ActionsProps) => {
-  const [visibleActions, setVisibleActions] = useState(visibleByDefault);
+  const [visibleActions, setVisibleActions] = useState(false);
 
   return (
     <StyledActionsWrapper
       onMouseEnter={() => setVisibleActions(true)}
-      onMouseLeave={() => setVisibleActions(visibleByDefault)}
+      onMouseLeave={() => setVisibleActions(false)}
```

Comment:
it doesn't break the logic for the left bar items?

Reply:
no